templates/subscription-centre/index.html

```diff
@@ -1,6 +1,213 @@
 {% extends "subscription-centre/base_subscription-centre.html" %}
 
 {% block content %}
```

Comment:
This template needs a title block, currently it's "| Ubuntu". "Subscription centre" should suffice.